contracts/tests/ocr2.spec.ts

```diff
@@ -914,7 +915,16 @@ describe("ocr2", async () => {
     await transmit(rounds+1, rounds+1, new BN(rounds+1), Buffer.from([0, 0, 0, 0, 0, 0, 0, 0]));
 
     // max value u64 juelsPerFeecoin
-    await transmit(rounds+2, rounds+2, new BN(rounds+2), Buffer.from([127, 127, 127, 127, 127, 127, 127, 127]));  
+    await transmit(rounds+2, rounds+2, new BN(rounds+2), Buffer.from([127, 127, 127, 127, 127, 127, 127, 127]));
+  })
+
+  it("TS client listens and parses state", async () => {
+    let cl = new Chainlink(program, provider);
+    // causes the tests cases to block finishing
+    // const feedNum = cl.onRound(state, (event) => {
+    //   console.log(event)
+    // })
+    let transmitTx = await transmit(16, 16, new BN(16));
```

Comment:
Can we use the SDK to test here? It would ensure that contract changes are in sync with the SDK. We already do this for the rust SDK by testing the hello-world example against the contracts in the tree.